components/mixins/profile/b2b-person-details.schema.json

```diff
@@ -26,6 +26,11 @@
           "description": "Field recording the current marketing/sales status of the Person.",
           "type": "string"
         },
+        "xdm:personSource": {
```

Comment:
Should this be an enum or set values? What is an example of the value that would be populated here?